DOCUMENTATION.md

```diff
@@ -171,7 +171,7 @@ The `task {}` CE is an independent project maintained by [Crowded](https://githu
 
 **IMPORTANT NOTICE**
 
-If you have `do!` bindings in your Giraffe web application then you must open the `FSharp.Control.Tasks.V2.ContextInsensitive` namespace to resolve any type inference issues:
+If you have `do!` bindings in your Giraffe web application then you must open the `FSharp.Control.Tasks.Builders` namespace to resolve any type inference issues:
```

Comment:
```suggestion
If you have `do!` bindings in your Giraffe web application then you must open the `FSharp.Control.Tasks` namespace to resolve any type inference issues:
```

Reply:
I see there are still a few other mentions of `FSharp.Control.Tasks.Builders` that can be `FSharp.Control.Tasks` instead, would you mind doing a replace all over the text?